### PR TITLE
Movie release fixes

### PR DESCRIPTION
**Pull request by Mussuranae**
Part of #5897 
Related to this [comment](https://github.com/blockframes/blockframes/issues/5897#issuecomment-853875664)

### CODE REVIEW

File: libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html [hunk 1]
```diff
@@ -1,22 +1,40 @@
 <section fxLayout="row" fxLayout.lt-sm="column" fxLayoutGap="24px">
   <ng-container *ngIf="(links | promotionalLinks: movie | async); let _links">
-    <ng-container *ngFor="let link of _links | slice:0:1">
-      <a mat-button [href]="link.url" target="_blank">
-        <mat-icon [svgIcon]="link.icon"></mat-icon>
-        <span class="mat-caption">{{ link.label }}</span>
-      </a>
+    <ng-container *ngIf="videos">
+      <button mat-button (click)="scrollToFooter()">
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html:4]: Ideally this should be managed by a fragment : `<a routerLink="." fragment="***">`

File: libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.ts [hunk 2]
```diff
@@ -7,9 +7,19 @@ import { Movie } from '@blockframes/movie/+state';
...
+  public videos = false;
 
+  ngOnInit() {
+    this.videos = this.movie.promotional.videos.otherVideos.some(video => video.storagePath);
+  }
+
+  scrollToFooter() {
+    const footer = document.getElementsByTagName('footer');
+    footer[0].scrollIntoView({ behavior: 'smooth' });
+    return;
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.ts:23]: no need to return.

File: libs/movie/src/lib/movie/form/start/start-tunnel.component.ts [hunk 3]
```diff
@@ -37,7 +37,7 @@ export class MovieFormStartTunnelComponent {
   async navigateToTunnel() {
     this.loadingTunnel.next(true);
     try {
-      const { id } = await this.movieService.create({productionStatus: predefinedTitleConfig[this.app], app: {[this.app]: { access: true}}});
+      const { id } = await this.movieService.create({...predefinedTitleConfig[this.app], app: {[this.app]: { access: true}}});
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/start/start-tunnel.component.ts:40]:
```typescript
const app = { [this.app]: { access: true} };
const { id } = await this.movieService.create({ ...predefinedTitleConfig[this.app], app });
```

**Review by GrandSchtroumpf — APPROVED** [review]

File: libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html (hunk 1 above)
Comment (Mussuranae) [reply libs/movie/src/lib/movie/components/promotional-links/promotional-links.component.html:4]: I tried, but it doesn't work I don't know why :/

**Pull request merged**